codalab/model/tables.py

```diff
@@ -93,6 +93,64 @@
     Column('parent_path', Text, nullable=False),
 )
 
+# Storage location for bundles.
+bundle_store = Table(
+    'bundle_store',
+    db_metadata,
+    Column(
+        'id',
+        BigInteger().with_variant(Integer, "sqlite"),
+        primary_key=True,
+        nullable=False,
+        autoincrement=True,
+    ),
+    Column('uuid', String(63), nullable=False),
+    # Usually root. For BYO bundle stores (not yet supported),
+    # it is the user who created the bundle store.
+    Column('owner_id', String(255), nullable=True),
+    # Name used to refer to this bundle store from the CLI.
```

Comment:
Make `name` nullable False

Reply:
We should do this to correspond with the worksheet table. Can you also check all other columns with worksheet to ensure they correspond in terms of things being nullable, etc.?

Reply:
owner_id on bundle worksheet is nullable=True

Reply:
matched everything with worksheet

Reply:
Oh, I guess that's kind of strange (having it nullable = True) but let's just keep it that way to be consistent